---
source: tests/snapshots.rs
assertion_line: 54
expression: output
---
success = false
stdout = ''
stderr = '''
Error:   × type mismatch
   ╭─[bad_array_type.rice:2:21]
 1 │ fn main() {
 2 │   let a: [string] = [1, 2] in
   ·                     ───┬──
   ·                        ╰── expected `[int]`, found `[string]`
 3 │   println(int_to_string(a[0]))
   ╰────

'''
